File: Sprint Web/week 4/src/Entity/Vote.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Vote
 *
 * @ORM\Table(name="vote", indexes={@ORM\Index(name="id_match", columns={"id_match"})})
 * @ORM\Entity()
 */
class Vote
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_vote", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idVote;

    /**
     * @var int
     * @ORM\ManyToOne(targetEntity="Matchs")
     *
     * @ORM\Column(name="id_match", type="integer", nullable=false)
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_match", referencedColumnName="id_match")
     * })
     */
    private $idMatch;

    /**
     * @var int
     *
     * @ORM\Column(name="id_user", type="integer", nullable=false)
     */
    private $idUser;

    /**
     * @var int
     *
     * @ORM\Column(name="vote", type="integer", nullable=false)

     */
    private $vote;

    /**
     * @return int
     */
    public function getIdVote(): int
    {
        return $this->idVote;
    }

    /**
     * @param int $idVote
     */
    public function setIdVote(int $idVote): void
    {
        $this->idVote = $idVote;
    }

    /**
     * @return int
     */
    public function getIdMatch(): int
    {
        return $this->idMatch;
    }

    /**
     * @param int $idMatch
     */
    public function setIdMatch(int $idMatch): void
    {
        $this->idMatch = $idMatch;
    }

    /**
     * @return int
     */
    public function getIdUser(): int
    {
        return $this->idUser;
    }

    /**
     * @param int $idUser
     */
    public function setIdUser(int $idUser): void
    {
        $this->idUser = $idUser;
    }



    /**
     * @return mixed
     */
    public function getVote()
    {
        return $this->vote;
    }

    /**
     * @param mixed $vote
     */
    public function setVote($vote): void
    {
        $this->vote = $vote;
    }


}

File: SprintMobile/src/com/gestionMatch/entities/Vote.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.gestionMatch.entities;

/**
 *
 * @author esprit
 */
public class Vote {
    private int idVote;
    private int idMatch;
    private int idUser;
    private int vote;

    public Vote() {
    }

    public Vote(int idVote, int idMatch, int idUser, int vote) {
        this.idVote = idVote;
        this.idMatch = idMatch;
        this.idUser = idUser;
        this.vote = vote;
    }

    public Vote(int idMatch, int idUser, int vote) {
        this.idMatch = idMatch;
        this.idUser = idUser;
        this.vote = vote;
    }

    public int getIdVote() {
        return idVote;
    }

    public void setIdVote(int idVote) {
        this.idVote = idVote;
    }

    public int getIdMatch() {
        return idMatch;
    }

    public void setIdMatch(int idMatch) {
        this.idMatch = idMatch;
    }

    public int getIdUser() {
        return idUser;
    }

    public void setIdUser(int idUser) {
        this.idUser = idUser;
    }

    public int getVote() {
        return vote;
    }

    public void setVote(int vote) {
        this.vote = vote;
    }

    @Override
    public String toString() {
        return "Vote{" + "idVote=" + idVote + ", idMatch=" + idMatch + ", idUser=" + idUser + ", vote=" + vote + '}';
    }
    
}
